### PR TITLE
Move submitted_motion_ids into meeting_user

### PR DESCRIPTION
**Pull request by reiterl**
Update model, data, tests and actions.

### CODE REVIEW

File: tests/system/action/meeting/test_import.py [hunk 1]
```diff
@@ -363,7 +363,6 @@ def get_user_data(self, obj_id: int, data: Dict[str, Any] = {}) -> Dict[str, Any
             "is_demo_user": False,
             "organization_management_level": None,
             "is_present_in_meeting_ids": [],
-            "submitted_motion_$_ids": [],
```
Comment (r-peschke) [inline tests/system/action/meeting/test_import.py:366]: Please move the removed templates field to an instance of meeting_user for import, testing also the import

File: openslides_backend/action/actions/motion_submitter/create.py [hunk 2]
```diff
@@ -41,24 +41,31 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
...
+            OrganizationManagementLevel.SUPERADMIN,
         ):
             assert_belongs_to_meeting(
                 self.datastore,
-                [fqid_from_collection_and_id("user", instance["user_id"])],
+                [fqid_from_collection_and_id("user", meeting_user["user_id"])],
                 meeting_id,
             )
 
         # check, if (user_id, motion_id) already in the datastore.
```
Comment (r-peschke) [inline openslides_backend/action/actions/motion_submitter/create.py:60]: user_id => meeting_user_id in comment

**Review by r-peschke — CHANGES_REQUESTED** [review]

File: tests/system/action/meeting/test_import.py (hunk 1 above)
Comment (reiterl) [reply tests/system/action/meeting/test_import.py:366]: There is a test for "personal_note_ids". I won't improve the tests here.

File: openslides_backend/action/actions/motion_submitter/create.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/actions/motion_submitter/create.py:60]: Removed the surplus comment.

**Review by r-peschke — APPROVED** [review]

**Pull request merged**